nservicebus/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -77,16 +77,16 @@ Consider the following example:
 
 To enable sending from `account_A` to `account_B`, following configuration has to be applied in the `account_A` endpoint
 
-snippet:AzureStorageQueueUseMultipleAccountNamesInsteadOfConnectionStrings1
+snippet:AzureStorageQueueUseMultipleAccountAliasesInsteadOfConnectionStrings1
 
 To enable sending from `account_B` to `account_A`, following configuration has to be applied in the `account_B` endpoint
 
-snippet:AzureStorageQueueUseMultipleAccountNamesInsteadOfConnectionStrings2
+snippet:AzureStorageQueueUseMultipleAccountAliasesInsteadOfConnectionStrings2
 
-`DefaultAccountName(string name)` maps a name of an account used by this endpoint.
-`AccountRouting().AddAccount(string accountName, string connectionString)` adds a mapping for other accounts, enabling using `@` notation for destination addresses `queue_name@accountName`.
+`DefaultAccountAlias(string alias)` maps an alias of an account used by this endpoint.
+`AccountRouting().AddAccount(string accountAlias, string connectionString)` adds a mapping for other accounts, enabling using `@` notation for destination addresses `queue_name@accountName`.
```

Comment:
AccountRouting().AddAccount is fragile to change. ie wont break based on compilable snippets. can we refer tot he concept and not the code?